Meadowlark-js/tests/http/local.33b.http

```diff
@@ -16,6 +16,22 @@ content-type: application/json
 
 ###
 @authToken1 = {{client1.response.body.$.access_token}}
+###
+
+### Authenticate client4 (this client has validation turned off, this is a replacement for reference-validation:false)
```

Comment:
After this PR, the "this is ..." comment won't make sense to readers.
```suggestion
### Authenticate client4 - assessment vendor for bypassing reference validation
```

Reply:
can do